lifetimes/utils.py

```diff
@@ -110,7 +110,11 @@ def find_first_transactions(transactions, customer_id_col, datetime_col, monetar
     if monetary_value_col:
         select_columns.append(monetary_value_col)
 
-    transactions = transactions[select_columns].sort(select_columns).copy()
+    # in python 3.6 with pandas 0.20.1 sort raise error, sort_values was added in 0.17.0
+    if int(pd.__version__.replace('.', '')) >= 170:
+        transactions = transactions[select_columns].sort_values(select_columns).copy()
+    else:
+        transactions = transactions[select_columns].sort(select_columns).copy()
```

Comment:
hm, in the requirements I explicitly ask for 0.19. Should we bump this to 0.20 and make this only sort_values?